modules/ROOT/pages/faq.adoc

```diff
@@ -110,3 +110,29 @@ To avoid this issue you can use the xref:quickstart.adoc#user-defined-schema[use
 Aura customers connecting from Databricks may encounter SSL handshake errors due to Databricks' custom Java security settings removing support for some encryption ciphers.
 
 See the Aura support article link:https://support.neo4j.com/s/article/1500003161121-Connecting-to-Aura-with-Databricks[Connecting to Aura with Databricks] for more information.
+
+== Troubleshooting
+
+=== How do I get the underlying Neo4j driver debug logs?
+
+[NOTE]
+====
+This is only possible with versions 4.1.4 and later of the Spark connector.
+====
+
+The Spark program needs to be configured for logging, as documented in https://spark.apache.org/docs/latest/configuration.html#configuring-logging[the Spark manual].
+
+Set the level to `TRACE` for the package `org.neo4j.driver`.
+It is also advised to set a higher level, like `WARN` or `ERROR`, for the packages under  `org.neo4j.driver.internal.shaded` to avoid getting too many logs.
+
+Log messages such as the following examples will then appear, offering a detailed view of the Bolt exchange between Neo4j's Bolt server and the Spark connector's Neo4j driver (format may vary based on your logging configuration):
+
+
+[source,logs]
```

Comment:
I don't think there is a highlighter called `logs`. Anyway, since it's just text, you can remove this and just leave the `----` delimiter.